cob_default_robot_config/cob4-1/head_joint_configurations.yaml

```diff
@@ -1 +0,0 @@
-../cob4-2/head_joint_configurations.yaml
```

Comment:
that's an earlier artefact. If the HW is changed now, we should update both (cob4-1 and cob4-2) with a 3dof head by using symlinks.